lib/glific/flows/flow_label.ex

```diff
@@ -58,6 +58,7 @@ defmodule Glific.Flows.FlowLabel do
     query =
       FlowLabel
       |> where([m], m.organization_id == ^organization_id)
+      |> select([m], %{uuid: m.uuid, name: m.name})
```

Comment:
This looks good, 

can you double check and confirm that we are not using this function on other places. 

```
FlowLabel.get_all_flowlabel
```

Reply:
Yes. Its good. We are using it in seeds and test files which expect the same result.